python/thunder/rdds/images.py

```diff
@@ -416,6 +416,85 @@ def crop(self, minbound, maxbound):
 
         return self._constructor(newrdd, dims=newdims).__finalize__(self)
 
+    def meanByRegions(self, selection):
+        """Collapses images into one or more spatial mean values as specified by the passed mask array or sequence of
```

Comment:
Suggested rewording: `Reduces images to one or more spatially averaged values using the given mask array or sequence of indices`